app/controllers/idv/in_person/usps_locations_controller.rb

```diff
@@ -26,7 +26,8 @@ def index
           city: search_params['city'], state: search_params['state'],
           zip_code: search_params['zip_code']
         )
-        response = proofer.request_facilities(candidate)
+        vector_of_trust = UspsInPersonProofing::EippHelper.extract_vector_of_trust(sp_session)
+        response = proofer.request_facilities(candidate, vector_of_trust)
```

Comment:
ditto about keyword args
```suggestion
        response = proofer.request_facilities(location: candidate, vector_of_trust:)
```